charts/couchdb/templates/statefulset.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apps/v1beta2
+apiVersion: {{ template "couchdb.statefulset.apiVersion" . }}
```

Comment:
Hardcode 'apps/v1' here instead. The latest couchdb helm chart uses 'apps/v1' and we don't need to support pre-1.15 k8s verisons.

Reply:
Fixed